serial_no: XUCAS1716015 #XUCAS1716011  
cam_name: "camera"
# auto_exposure: true
exposure_time: 30000
# gain: 1
# auto_exposure_priority: 0.8
auto_wb: true
yaml_url: "package://ximea_camera/calibration/ost.yaml"
image_data_format: "XI_RAW8"  #XI_RGB24, XI_RAW8
camera_frame: "camera_optical"
#height: 700
#width: 1000
binning_enabled: true
downsample_factor: 2


